src/__tests__/volume.test.ts

```diff
@@ -263,6 +263,17 @@ describe('volume', () => {
         expect(stat.isDirectory()).toBe(true);
         expect(vol.readdirSync('/dir')).toEqual([]);
       });
+
+      it('Files as buffer', () => {
+        const vol = new Volume();
+        const json = {
+          '/buffer': Buffer.from('bip-boup', 'utf-8'),
+        };
+        vol.fromJSON(json);
+        expect(vol.toJSON()).toEqual({
```

Comment:
```suggestion
        expect(vol.toJSON()).toStrictEqual({
```

(I know we're not using it in other tests, but we should use it going forward)

Reply:
Done.